source/index.html.md.erb

```diff
@@ -388,6 +388,63 @@ This core hierarchy of Sheet > Column > Row > Cell is essential to working with
 
 # Sample Code and SDKs
 
+> Install the SDK
+
+```javascript
+npm install smartsheet
+```
+
+```csharp
+Install-Package smartsheet-csharp-sdk
```

Comment:
In the Visual Studio Package Manager Console:

Reply:
Done